Contributors_Guide.md

```diff
@@ -0,0 +1,11 @@
+Contributors Guide
```

Comment:
Such file is usually called CONTRIBUTING.md: https://github.com/blog/1184-contributing-guidelines